openeo-geotrellis/src/main/scala/org/openeo/geotrellis/package.scala

```diff
@@ -89,4 +90,32 @@ package object geotrellis {
 
     Region.of(regionName)
   }
+
+  def toSigned(cellType: CellType): CellType = {
+    cellType match {
+      case UByteCellType => ByteCellType
+      case UByteConstantNoDataCellType => ByteConstantNoDataCellType
+      case c: UByteUserDefinedNoDataCellType => ByteUserDefinedNoDataCellType(c.noDataValue)
+      case UShortCellType => ShortCellType
+      case UShortConstantNoDataCellType => ShortConstantNoDataCellType
+      case c: UShortUserDefinedNoDataCellType => ShortUserDefinedNoDataCellType(c.noDataValue)
+      case FloatConstantNoDataCellType => cellType
+      case ShortConstantNoDataCellType => cellType
+      case BitCellType => cellType
+      case ByteConstantNoDataCellType => cellType
+      case ByteCellType => cellType
+      case _: ByteUserDefinedNoDataCellType => cellType
```

Comment:
Or to align with the rest:

```scala
case ByteUserDefinedNoDataCellType(_) => cellType
```

Reply:
Changed too. And added `ByteUserDefinedNoDataCellType` to the `toSigned` test